be/src/olap/olap_cond.h

```diff
@@ -46,7 +46,8 @@ enum CondOp {
     OP_GE = 5,      // greater or equal
     OP_IN = 6,      // IN
     OP_IS = 7,      // is null or not null
-    OP_NULL = 8    // invalid OP
+    OP_NULL = 8,     // invalid OP
+    OP_NOT_IN = 9    // not in
```

Comment:
Normally, the invalid OP should be at the beginning(-1) or at last of the enum.
I am not sure if we persist the order of CondOp in storage. If not, I think its better to move the `OP_NULL` to the beginning and set it as -1.